src/redux/books/books.js

```diff
@@ -0,0 +1,44 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [
```

Comment:
Please set the empty array as the initial state.

Reply:
Changed.